smithy-cli/src/main/java/software/amazon/smithy/cli/commands/CleanCommand.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.smithy.cli.commands;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.logging.Logger;
+import software.amazon.smithy.build.SmithyBuild;
+import software.amazon.smithy.build.model.SmithyBuildConfig;
+import software.amazon.smithy.cli.ArgumentReceiver;
+import software.amazon.smithy.cli.Arguments;
+import software.amazon.smithy.utils.IoUtils;
+import software.amazon.smithy.utils.ListUtils;
+
+final class CleanCommand extends SimpleCommand {
+
+    private static final Logger LOGGER = Logger.getLogger(CleanCommand.class.getName());
+
+    CleanCommand(String parentCommandName) {
+        super(parentCommandName);
+    }
+
+    @Override
+    protected List<ArgumentReceiver> createArgumentReceivers() {
+        return ListUtils.of(new ConfigOptions());
+    }
+
+    @Override
+    public String getName() {
+        return "clean";
+    }
+
+    @Override
+    public String getSummary() {
+        return "Removes Smithy build artifacts";
```

Comment:
nit
```suggestion
        return "Removes Smithy build artifacts.";
```

Reply:
We are inconsistent with ending in a period, so I'll add a period to all command summaries.